L/libletsberational/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LetsBeRational"
```

Comment:
Note that this name must match exactly the name of the directory

Reply:
Ok, this is the name I'd actually like so I will change the directory.